src/libint2_wrappers/libint2_wrappers.h

```diff
@@ -82,7 +82,7 @@ void parallel_do(Lambda& lambda, int nthreads) {
 }
 #else  // use C++11 threads
 std::vector<std::thread> threads;
-for (int thread_id = 0; thread_id != libint2::nthreads; ++thread_id) {
+for (int thread_id = 0; thread_id != nthreads; ++thread_id) {

```

Comment:
can you keep the libint2 namespace and still compile it successfully?

Reply:
It is a good question since I'm using some tricks to avoid libint2 compilation :)

Looks like this code was stolen from libint2 project:
https://github.com/evaleev/libint/blob/2ca75836f139226ecf49cf8f4a9434cd3e8b1735/include/libint2/lcao/1body.h#L178-L186

Reply:
not from this place - i think i used a different example from some libint2;